dockerfiles/itest/itest/puppet_services.d/service_one

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We discussed in person and think it is best to add these as puppet services otherwise we have to rewrite a bunch of tests. We considered also just removing the classic services but this is the majority of the tests because we can't test paasta services without mesos etc.